README.md

```diff
@@ -1,3 +1,146 @@
+# Juror Backend API
 
-# Juror Digital - API
+This is the API for the Juror Modernisation project. It is a Spring Boot application that provides RESTful endpoints for
+managing jurors.
 
+# Building and deploying the application
+
+## Prerequisites
+
+- [Java 17](https://www.oracle.com/java)
+- [Docker](https://www.docker.com)
+
+### Environment variables
+
+The following application settings are required to run the application. They can be set in the 
+`application-local.yaml` file or as environment variables. The application will look for the environment variables first.
+
+```
+DB_PASSWORD=postgres
+DB_URL=jdbc:postgresql://localhost:5432/juror
+DB_USERNAME=juror
+JWT_SECRET_BUREAU=
+JWT_SECRET_HMAC=
+JWT_SECRET_PUBLIC=
+NOTIFY_KEY=
+NOTIFY_REGION_KEY_1=
+NOTIFY_REGION_KEY_2=
+NOTIFY_REGION_KEY_3=
+NOTIFY_REGION_KEY_4=
+NOTIFY_REGION_KEY_5=
+NOTIFY_REGION_KEY_6=
+NOTIFY_REGION_KEY_7=
```

Comment:
Do we want to include some none-secure defaults here?

Reply:
followed what Darts did and left it blank..